ipywidgets/widgets/widget.py

```diff
@@ -223,6 +272,10 @@ def open(self):
             if buffers:
                 # FIXME: workaround ipykernel missing binary message support in open-on-init
                 # send state with binary elements as second message
+                # TODO: if this gets fixed, _split_state_buffers does not need to have a seperate
```

Comment:
Have you tried it? It seems that Comm supports buffers in the init: https://github.com/ipython/ipykernel/blob/master/ipykernel/comm/comm.py#L50